### ISSUE TITLE
[bug]: Focus ring not visible for breadcrumbs

### ISSUE THREAD

**Issue by luke-denton-aligent** [magento/pwa-studio#3484]
**Describe the bug**

Keyboard users will have a great deal of trouble determining when they're focused on breadcrumbs due the edges of the focus ring being cut off

**To reproduce**

1. Go to https://venia.magento.com/default/juno-sweater.html
2. Click on the text "Juno Sweater" in the breadcrumbs (this isn't a link)
3. Press Shift + TAB
5. TAB/Shift + TAB through the breadcrumbs to see the issues of the focus ring being cut off

**Expected behavior**

In order to preserve accessibility, a user should always be able to clearly see which element is in focus

**Screenshots**

![image](https://user-images.githubusercontent.com/18253422/137039758-898f11e5-f5ec-4ae9-bbb0-380698861e8a.png)

**Possible solutions**

Make the containing element of the Breadcrumbs larger.
Rather than using `margin` to position the element, we should use `padding`.
We'll then need to address the hard coded height of the element, to either factor in the added padding, or otherwise remove the height in favour of another placeholder method

**Debug Report**

Command not found

**Please complete the following device information:**

-   Device [e.g. iPhone6, PC, Mac, Pixel3]: Desktops
-   Browser [e.g. Chrome, Safari]: All
-   Browser Version [e.g. 22]: All
-   Magento Version [e.g Magento Commerce 2.4]: N/A

**Please let us know what packages this bug is in regards to:**

-   [x] `venia-ui`

**Comment by supernova-at** [comment]
@magento export issue to JIRA project PWA as Bug
